src/main/java/frc/robot/commands/Climb.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.RobotConfig.ClimberConfig;
+import frc.robot.subsystems.climber.Climber;
+
+public class Climb extends Command {
+  private final Climber m_climber;
+  private boolean m_reverse;
+
+  public Climb(Climber climber, boolean reverse) {
+    m_climber = climber;
+    m_reverse = reverse;
+
+    addRequirements(climber);
+  }
+
+  @Override
+  public void initialize() {
+    int multiplier = m_reverse ? -1 : 1;
+    m_climber.setMotorSpeed(ClimberConfig.kDefaultSpeed*multiplier);
+  }
+
+
+  @Override
+  public void end(boolean interrupted) {
+    m_climber.setMotorSpeed(ClimberConfig.kStallInput);
```

Comment:
I get the feeling you should use PID here to keep us up.... you could also make a function called `hold()` in the Climber subsystem which accomplishes this.

Reply:
The climber actually does not need PID. It holds itself up because of gear ratios and a physical break.